examples/spark-viz/build.sbt

```diff
@@ -39,7 +39,7 @@ val SparkCompatibleVersion = "3.0"
 
 val HadoopVersion = "2.7.2"
 
```

Comment:
SeondaViz package is no longer available. The correct dependency should be `sedona-spark-3.0`